Resources/Prototypes/Entities/Structures/Machines/ftldrive.yml

```
@@ -0,0 +1,37 @@
+  placement:
+    mode: SnapgridCenter
+  components:
+    - type: Transform
+      anchored: true
+    - type: Physics
+      bodyType: Static
+    - type: Sprite
+      sprite: Structures/Power/Generation/ame.rsi
+      snapCardinals: true
+      layers:
+      - state: control
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)